components/byline/index.json-ld

```diff
@@ -0,0 +1,204 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Institutional byline",
+		"fr": "Institution responsable"
+	},
+	"description": {
+		"en": "The institutional byline provides people with a link to the institution or institutions responsible for the content.",
+		"fr": "La signature institutionnelle fournit aux personnes un lien vers l’institution ou les institutions responsables du contenu."
+	},
+	"modified": "2024-12-10",
+	"componentName": "byline",
+	"status": "demoted",
+	"version": "1.1.0",
+	"pages": {
+		"docs": [
+			{
+				"title": "Institutional byline",
+				"language": "en",
+				"path": "byline-doc-en.html"
+			},
+			{
+				"title": "Institution responsable",
+				"language": "fr",
+				"path": "byline-doc-fr.html"
+			}
+		]
+	},
+	"a11yGuidance": "No accessibility guidance.",
+	"variations": [
+		{
+			"name": {
+				"en": "Institutional byline - default",
+				"fr": "Institution responsable - par défaut"
+			},
+			"status": "demoted",
+			"description": {
+				"en": "The institutional byline provides people with a link to the institution or institutions responsible for the content.",
+				"fr": "La signature institutionnelle fournit aux personnes un lien vers l’institution ou les institutions responsables du contenu."
+			},
+			"iteration": "_:iteration_byline_2",
+			"example": [
+				{
+					"en": { "href": "../../templates/content-page/content-en.html", "text": "Content page (byline after the main title)" },
+					"fr": { "href": "../../templates/content-page/content-fr.html", "text": "Page de contenu (institution responsable sous le titre principal)" }
+				}
+			],
+			"implementation": [
+				"_:implement_byline"
+			],
+			"history": [
+				{
+					"en": "December 2024 - Deprecation of the component - Moved to the Main title component. Use the Contributors component instead.",
+					"fr": "Décembre 2024 - Composante maintenant obsolète - Déplacée vers la componante Titre principal. Utilisez plutôt la composante Contributeurs."
+				},
+				{
+					"en": "April 2016 - Addition of the class <code>.gc-byline</code>.",
+					"fr": "Avril 2016 - Ajout de la classe <code>.gc-byline</code>."
+				},
+				{
+					"en": "WET-BOEW 4 - Initial implementation of the component.",
+					"fr": "WET-BOEW 4 - Implémentation initiale pde la composante."
```

Comment:
To do: Adjust the date to match the documented iteration.

En fait, il a été initialement ajouté en 2016 avec le gabarit "Complex Service" et qui est devenu très rapidement le gabarit "Advanced service".